parenthetics.py

```diff
@@ -0,0 +1,17 @@
+"""Parenthetics module."""
+from link_list import link_list
```

Comment:
where is the file you're importing `link_list` from? It's not a part of your repository.